### PR TITLE
lightning: add using lighting to migrate aurora data tutorial

### PR DESCRIPTION
**Pull request by TomShawn**
### What is changed, added or deleted? (Required)

Add a tutorial for migrating aurora data by lightning

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/5287
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by glorv** [comment]
LGTM

File: migrate-from-aurora-using-lightning.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+    # Uses single quoted strings to avoid escaping.
+    pattern = '(?i)^(?:[^/]*/)*([a-z0-9_]+)\.([a-z0-9_]+)/(?:[^/]*/)*(?:[a-z0-9\-_.]+\.(parquet))$'
+    schema = '$1'
+    table = '$2'
+    type = '$3'
+    ```
+
+> **Note:**
+>
+> - The example in this document uses the Local-backend for best performance. You can also choose TiDB-backend or Importer-backend according to your need. For detailed introduction of the three backend modes, see [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md).
```
Comment (CharLotteiu) [inline migrate-from-aurora-using-lightning.md:69]:
```suggestion
> - The above example uses the Local-backend for best performance. You can also choose TiDB-backend or Importer-backend according to your need. For detailed introduction of the three backend modes, see [TiDB Lightning Backends](/tidb-lightning/tidb-lightning-backends.md).
```

File: migrate-from-aurora-using-lightning.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+> - Because the path for exporting snapshot data from Aurora is different from the default file naming format supported by TiDB Lightning, you need to set additional file routing configuration.
+> - If TLS is enabled in the target TiDB cluster, you also need to configure TLS.
+
+For other configuration, see [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
```
Comment (CharLotteiu) [inline migrate-from-aurora-using-lightning.md:73]:
```suggestion
For other configurations, see [TiDB Lightning Configuration](/tidb-lightning/tidb-lightning-configuration.md).
```

**Review by CharLotteiu — COMMENTED** [review]
rest LGTM.

File: migrate-from-aurora-using-lightning.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+    > - Set the parameters of the data source address and the path of output files according to your actual situation.
+    > - If you need export all database tables, you do not need to set the `--filter` parameter. If you only need to export some of the database tables, configure `--filter` according to [table-filter](https://github.com/pingcap/tidb-tools/blob/master/pkg/table-filter/README.md).
+
+2. Use TiDB Lightning to create table schemas:
+
+    ```
+    ./tidb-lightning -config tidb-lightning.toml -d ./schema -no-schema=false
+    ```
+
+    In this example, TiDB Lightning is only used to create table schemas, so you need to execute the above command quickly. At a regular speed, ten table creation statements can be executed in one second.
```
[the remark this answers is not in the document]
Comment (glorv) [reply migrate-from-aurora-using-lightning.md:96]: here we just want to express that "this command can be executed very quickly", the original Chinese description is not accurate, so sorry for that.

[the remark this answers is not in the document]
Comment (TomShawn) [reply migrate-from-aurora-using-lightning.md:96]: According to @glorv, "应迅速执行以上命令" means that "TiDB Lightning can quickly process the command".
```suggestion
    In this example, TiDB Lightning is only used to create table schemas and can quickly process the above command. At a regular speed, ten table creation statements can be executed in one second.
```

**Review by CharLotteiu — APPROVED** [review]

**Pull request merged**